benchmarks/hash_table/static_map/contains_bench.cu

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2021-2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <defaults.hpp>
+#include <distribution.hpp>
+#include <key_generator.hpp>
+
+#include <cuco/static_map.cuh>
+
+#include <nvbench/nvbench.cuh>
+
+#include <thrust/device_vector.h>
+#include <thrust/iterator/discard_iterator.h>
+
+using namespace cuco::benchmark;
+using namespace cuco::benchmark::defaults;
+
+/**
+ * @brief A benchmark evaluating `contains` performance:
+ */
+template <typename Key, typename Value, typename Dist>
+std::enable_if_t<(sizeof(Key) == sizeof(Value)), void> static_map_contains(
+  nvbench::state& state, nvbench::type_list<Key, Value, Dist>)
+{
+  auto const num_keys      = state.get_int64_or_default("NumInputs", defaults::N);
+  auto const occupancy     = state.get_float64_or_default("Occupancy", defaults::OCCUPANCY);
+  auto const matching_rate = state.get_float64_or_default("MatchingRate", defaults::MATCHING_RATE);
```

Comment:
```suggestion
  auto const num_keys      = state.get_int64_or_default("NumInputs", N);
  auto const occupancy     = state.get_float64_or_default("Occupancy", OCCUPANCY);
  auto const matching_rate = state.get_float64_or_default("MatchingRate", MATCHING_RATE);
```

Just a suggestion (also how I did this for the multimap benchmarks). I do find your approach more expressive.